packages/consent/consent-wrapper-onetrust/src/domain/wrapper.ts

```diff
@@ -11,10 +11,18 @@ import {
   getConsentedGroupIds,
   getOneTrustGlobal,
 } from '../lib/onetrust-api'
-
+import { coerceConsentModel } from './consent-model'
 export interface OneTrustSettings {
   integrationCategoryMappings?: CreateWrapperSettings['integrationCategoryMappings']
   disableConsentChangedEvent?: boolean
+  /**
+   * Override configured consent model
+   * - opt-in  (default) - load segment and all destinations without waiting for explicit consent.
+   * - opt-out (strict/GDPR) - wait for explicit consent before loading segment
```

Comment:
Aren't these flipped? `opt-in` is the one where we wait for you to opt-in before loading, and opt-out is where we load until you explicitly opt out?

Reply:
Ooh, yep, the README is correct but these comments are flipped. Good catch.